issuance/cert.go

```diff
@@ -27,9 +27,21 @@ import (
 
 // ProfileConfig describes the certificate issuance constraints for all issuers.
 type ProfileConfig struct {
+	// AllowMustStaple, when false, causes all IssuanceRequests which specify the
+	// OCSP Must Staple extension to be rejected.
 	AllowMustStaple bool
-	AllowCTPoison   bool
-	AllowSCTList    bool
+	// AllowCTPoison, when false, causes all IssuanceRequests which want the
+	// CT Poison extension to be rejected.
+	// Deprecated: We will always allow the CT Poison extension because it is
+	// mandated for Precertificates. This boolean has no effect.
```

Comment:
A bit funny to explain what the flag does, then explain that it does nothing.

```suggestion
	// AllowCTPoison has no effect.
	// Deprecated: We will always allow the CT Poison extension because it is
	// mandated for Precertificates.
```